endian_generic.go

```diff
@@ -1,3 +1,4 @@
+//go:build !mips && !mips64 && !ppc64 && !s390x && !amd64 && !386 && !arm && !arm64 && !mipsle && !mips64le && !ppc64le && !riscv64 && !wasm
```

Comment:
I think this now needs an `&& !loong64` clause.

Reply:
Right, done!